mkosi/__init__.py

```diff
@@ -1692,15 +1692,17 @@ def gzip_binary(context: Context) -> str:
 
 
 def fixup_vmlinuz_location(context: Context) -> None:
-    for d in context.root.glob("boot/vmlinuz-*"):
-        kver = d.name.removeprefix("vmlinuz-")
-        vmlinuz = context.root / "usr/lib/modules" / kver / "vmlinuz"
-        # Some distributions (OpenMandriva) symlink /usr/lib/modules/<kver>/vmlinuz to /boot/vmlinuz-<kver>, so get rid
-        # of the symlink and copy the actual vmlinuz to /usr/lib/modules/<kver>.
-        if vmlinuz.is_symlink() and vmlinuz.is_relative_to("/boot"):
-            vmlinuz.unlink()
-        if not vmlinuz.exists():
-            shutil.copy2(d, vmlinuz)
+    # Some architectures ship an uncompressed vmlinux (ppc64el, riscv64)
+    for type in "vmlinuz", "vmlinux":
```

Comment:
Let's not have this be an implicit tuple
```suggestion
    for type in ("vmlinuz", "vmlinux"):
```